hack/functest.sh

```diff
@@ -4,4 +4,4 @@ set -xe
 
 source ./cluster/kubevirtci.sh
 
-KUBECONFIG=${KUBECONFIG:-$(kubevirtci::kubeconfig)} $GO test ./tests/... $E2E_TEST_ARGS -timeout=40m -ginkgo.v -test.v -race --test-suite-params="$POLARION_TEST_SUITE_PARAMS"
+KUBECONFIG=${KUBECONFIG:-$(kubevirtci::kubeconfig)} $GO test ./tests/... $E2E_TEST_ARGS -test.timeout=40m -ginkgo.v -test.v --test-suite-params="$POLARION_TEST_SUITE_PARAMS"
```

Comment:
Why do we remove `-race` ?

Reply:
I got an error [running](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/655/pull-e2e-cluster-network-addons-operator-kubemacpool-functests/1328573319720472576#1:build-log.txt%3A1017) with it on the CNAO, saying it's not a valid flag anymore.
si I looked at go test [manual](https://golang.org/pkg/cmd/go/internal/test/) - the flag is not there anymore, 
plus, all other CNAO repos don't use it.

Reply:
It's from ginkgo so it has to be `-ginkgo.race` but since this is not unit tests I don't think we need it.

Reply:
there is no `-ginkgo.race` flag..